services/chrome-web-store/chrome-web-store-price.tester.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const Joi = require('joi')
+const { ServiceTester } = require('../tester')
+
+const t = (module.exports = new ServiceTester({
```

Comment:
Could these use `createServiceTester()`?

Reply:
Yeah good question. I thought Version and Price should be able to use that because both of those only export a single integration, but if I switch Version and Price to use `const t = (module.exports = require('../tester').createServiceTester())` then if I run `npm run test:services -- --only=chrome-web-store` it only runs the tests for Rating and Downloads (although I can run `npm run test:services -- --only=chromewebstoreversion` and `npm run test:services -- --only=chromewebstoreprice` to individually run the version/price tests).

I figured it is better to retain the property that `npm run test:services -- --only=chrome-web-store` runs the whole test suite so I left them all manually creating a `ServiceTester`. I haven't looked into it in any more detail though. Any thoughts on this?

Reply:
If you use`{ id: 'ChromeWebStorePrice', pathPrefix: '/chrome-web-store/p' }` I think that should give the result you're looking for. That way, everything aligns with the `createServiceTester()` naming convention and the way to run them all becomes `--only=chromewebstore` or `--only=ChromeWebStore`.

We should look at the ServiceTester interface and try to align it a bit better with BaseService, maybe by changing `pathPrefix` to `base`, removing its leading `/`, not defaulting to the value of `id`, and also renaming `id` to `name`.